plugins/woocommerce/client/admin/client/customize-store/assembler-hub/toolbar/shuffle.tsx

```diff
@@ -79,7 +80,9 @@ export default function Shuffle( { clientId }: { clientId: string } ) {
 		patternName: string;
 	} = useSelect(
 		( select ) => {
-			const { getBlockAttributes } = select( blockEditorStore );
+			const { getBlockAttributes } = select( blockEditorStore ) as {
+				getBlockAttributes: ( clientId: string ) => BlockAttributes;
+			};
```

Comment:
Should we move some of these `blockEditorStore` to a central place, like a types file? and use them here. Given we manually set these in a couple places, like here: https://github.com/woocommerce/woocommerce/blob/843187ceff692bb78d59e3110a436f7213805f4a/plugins/woocommerce/client/admin/client/customize-store/assembler-hub/auto-block-preview.tsx#L131-L133

Reply:
As we decided, I added a comment (4fdd78f).